src/plugins/chart_expressions/expression_xy/public/components/tooltip/tooltip.scss

```diff
@@ -22,9 +22,14 @@
   }
 }
 
+.detailedTooltip__labelContainer {
+  max-width: $euiSizeXL * 5;
+}
+
 .detailedTooltip__labelContainer,
 .detailedTooltip__valueContainer {
   overflow-wrap: break-word;
+  word-break: break-all;
   word-wrap: break-word;
```

Comment:
I believe `word-wrap` is just an alias for `overflow-wrap`, so I think it can be safely removed here.

```suggestion
```